.github/workflows/ci.yml

```diff
@@ -15,12 +15,12 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macos-latest
+            #- macos-latest
```

Comment:
is this intended in this PR?

Reply:
yes, macos building was failing. I think we should disabled it until we can make it work again (or remove it)